Add conflig flow for Influxdb integration (#43598)
home-assistant/home-assistant.github.io: source/_integrations/influxdb.markdown

The configuration differs between versions. The documentation below notes when fields apply to specific versions.

{% include integrations/config_flow.md %}

> 

Reviewer: ```suggestion

If you have an existing YAML configuration, it will be imported automatically as a config entry. Additional options such as entity filters, tags, and measurement attributes are configured via YAML. For details, refer to the [configuration](#configuration) section below.

```

> ### InfluxDB 1 Configuration

Reviewer: ```suggestion
### InfluxDB 1.x configuration options
```

{% configuration_basic %}
URL:

>     description: "The URL to the InfluxDB Host."

Reviewer: ```suggestion
    description: "The URL of your InfluxDB host."
```

Username:
    description: "Database username with read/write privileges."
Password:
    description: "Password for the database user."
Database:
    description: "Database name. You must create the database before you can write to it."
Verify SSL:

>     description: "Verify SSL certificate for HTTPS request."

Reviewer: ```suggestion
    description: "Verify SSL certificate for HTTPS requests."
```

SSL CA Certificate:

>     description: "Optional path of a CA certificate to be used during SSL verification."

Reviewer: ```suggestion
    description: "Upload an optional CA certificate to use for SSL verification. Accepted formats: `.pem`, `.crt`, `.cer`, `.der`."
```

{% endconfiguration_basic %}

> ### InfluxDB 2 and 3 Configuration

Reviewer: ```suggestion
### InfluxDB 2.x and 3 configuration options
```

{% configuration_basic %}
URL:

>     description: "The URL to the InfluxDB Host."

Reviewer: ```suggestion
    description: "The URL of your InfluxDB host."
```

Organization:
    description: "Organization ID. For InfluxDB 2.x, find this in your installation URL after /orgs. For InfluxDB 3, this value is required but not validated—use any value."
Bucket:
    description: "For InfluxDB 2.x, the bucket name. For InfluxDB 3, this maps to the database name."
Token:
    description: "Auth token with write access."
Verify SSL:

>     description: "Verify SSL certificate for HTTPS request."

Reviewer: ```suggestion
    description: "Verify SSL certificate for HTTPS requests."
```

SSL CA Certificate:

>     description: "Optional path of a CA certificate to be used during SSL verification."

Reviewer: ```suggestion
    description: "Upload an optional CA certificate to use for SSL verification. Accepted formats: `.pem`, `.crt`, `.cer`, `.der`."
```
